geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/index/GeoMesaSchemaValidator.scala

```diff
@@ -120,3 +121,13 @@ object IndexConfigurationCheck {
     require(sft.getAttributeShards > 0 && sft.getAttributeShards < 128, "Attribute shards must be between 1 and 127")
   }
 }
+
+object PropertyCheck {
+  def validateDuplicateProperty(sft: SimpleFeatureType):Unit = {
+    val names = sft.getAttributeDescriptors.asScala.map(descriptor => descriptor.getLocalName)
+    val duplicateProperties = names.diff(names.distinct)
+    if ( duplicateProperties.nonEmpty ){
+      throw new IllegalArgumentException(s"duplicate attr: '${duplicateProperties.mkString(",")}' ")
```

Comment:
Could you make the error message a little clearer?

```suggestion
      throw new IllegalArgumentException(s"Schema contains duplicate attribute names: '${duplicateProperties.mkString(", ")}' ")
```